cgi-bin/DW/Controller/Inbox.pm

```diff
@@ -0,0 +1,659 @@
+package DW::Controller::Inbox;
+use strict;
```

Comment:
This file is missing the header copyright notices and also the normal preamble (use v5.10 etc).